client/indexer-db/migrations/2024-09-26-145832_create_bsp/up.sql

```diff
@@ -4,7 +4,8 @@ CREATE TABLE bsp (
     account VARCHAR NOT NULL,
     capacity NUMERIC(20, 0) NOT NULL,
     created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
-    updated_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP
+    updated_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    stake NUMERIC(38, 0) NOT NULL DEFAULT 0
```

Comment:
Maybe switch the order of fields so that `created_at` and `update_at` are the last ones?